components/support/webextensions/src/test/java/mozilla/components/support/webextensions/WebExtensionSupportTest.kt

```diff
@@ -404,6 +404,46 @@ class WebExtensionSupportTest {
         assertEquals(popupSessionId, (actionCaptor.value as TabListAction.RemoveTabAction).tabId)
     }
 
+    @Test
+    fun `reacts to action popup being toggled by onActionPopupToggleOverride`() {
```

Comment:
Looking at other tests, we should change the name here to `allows overriding action popup behaviour` or similar. That makes it clearer what we're testing here.